formats/filtered/src/View/MapView.php

```diff
@@ -78,11 +77,6 @@ public function getJsDataForRow( ResultItem $row ) {
 						}
 					}
 
-				} else {
-					$this->getQueryPrinter()->addError(
-						Message::newFromKey( 'srf-filtered-map-geocoordinateparser-missing-error' )->inContentLanguage(
```

Comment:
I think this is the only place where this message is used, so it can be removed from the i18n files.

Reply:
Good catch

Reply:
https://github.com/SemanticMediaWiki/SemanticResultFormats/commit/561384af15a33f5e16a235cf0f80d96fd04ed548